apps/ftu-ui/src/assets/TEMP/ftu-cell-summaries.jsonld

```diff
@@ -6745,6 +6747,7 @@
     {
       "@type": "CellSummary",
       "cell_source": "https://purl.humanatlas.io/2d-ftu/kidney-cortical-collecting-duct",
+      "dataset_id": "https://cns-iu.github.io/hra-cell-type-populations-supporting-information/data/enriched_rui_locations.jsonld#36e76662-60b8-4193-8a70-1bfd4f6938d0_D088_Lung",
```

Comment:
Datasets should have their own cell summary. For instance, this dataset_id would have another "CellSummary" object with cell_source equal to the dataset it.